cmd/agent/app/status.go

```diff
@@ -49,32 +52,37 @@ var statusCmd = &cobra.Command{
 	},
 }
 
+var componentCmd = &cobra.Command{
+	Use:   "component",
+	Short: "Print the component status",
+	Long:  ``,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := common.SetupConfigWithoutSecrets(confFilePath)
+		if err != nil {
+			return fmt.Errorf("unable to set up global agent configuration: %v", err)
+		}
+		if flagNoColor {
+			color.NoColor = true
+		}
+
+		if len(args) != 1 {
```

Comment:
Detail but seeing the usage:
```
	Use:   "status [component [name]]",
```
there is two parameters possible, so it should be `< 2`.

Reply:
Whoops, got it, forget that.